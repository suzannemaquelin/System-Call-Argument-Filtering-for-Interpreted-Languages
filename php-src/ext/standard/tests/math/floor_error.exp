*** Testing floor() :  error conditions ***

-- Too many arguments --

Warning: floor() expects exactly 1 parameter, 2 given in %s on line %d
NULL

-- Too few arguments --

Warning: floor() expects exactly 1 parameter, 0 given in %s on line %d
NULL
===Done===