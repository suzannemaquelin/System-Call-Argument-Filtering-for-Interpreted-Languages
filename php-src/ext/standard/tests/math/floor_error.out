Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
*** Testing floor() :  error conditions ***

-- Too many arguments --

Warning: floor() expects exactly 1 parameter, 2 given in /root/php-src/ext/standard/tests/math/floor_error.php on line 12
NULL

-- Too few arguments --

Warning: floor() expects exactly 1 parameter, 0 given in /root/php-src/ext/standard/tests/math/floor_error.php on line 15
NULL
===Done===